backends/platform/sdl/sdl-window.cpp

```diff
@@ -265,6 +264,16 @@ SDL_Surface *copySDLSurface(SDL_Surface *src) {
 	return res;
 }
 
+int SdlWindow::getDisplayIndex() const {
+	if (_window) {
+		int displayIndex = SDL_GetWindowDisplayIndex(_window);
```

Comment:
This is an SDL2 call. The equivalent SDL1 call was SDL_GetCurrentVideoDisplay(). There should be a version check here, so that this compiles with SDL1

Reply:
The version check is at line 245. Since this function is only used in SDL2 there's no need for a SDL1 version.

Reply:
Indeed. That SDL2 block is quite large, and includes other functions too.